about.html

```diff
@@ -0,0 +1,181 @@
+        </div>
+    </section>
+      <!-- Logo section -->
+    <section class="about-the-logo mb-3 py-3">
+        <div class="atl-container">
+            <h3 class="atl-header">Digital Transformation African Summit 2023 Logo</h3>
+            <hr class="hr-line">
+            <p class="atl-description mb-4 py-2">
+                The Digital Transformation African Summit 2023 Logo represents the essence and vision of the upcoming summit, which focuses on driving digital transformation across the African continent.
+                 This logo incorporates elements that symbolize innovation, progress, and the unity of African nations in embracing the digital future.<span class="none">
+                The central element of the logo is a sleek and modern icon representing a stylized African continent.
+                This icon is crafted with clean lines and geometric shapes, reflecting a contemporary design aesthetic.
+                It embodies the forward-thinking mindset of the summit and signifies the collective efforts of African countries in embracing digital transformation.</span>
+            </p>
+            <img src="img/abt-logo.png" alt="" class="atl-logo-img py-4">
+        </div>
+    </section>
+    <hr>
+           <!-- Past summit -->
+    <section class="see-past mb-3 py-3">
+        <div class="sp-container">
+            <h3 class="atl-header">See past the CS African Summits</h3>
+            <hr class="hr-line">
+            <p class="sp-description mb-3">
+                Take a look at the last two CS African Summits which took place in Nairobi Kenya and Pretoria South Africa 
+            </p>
+            <div class="summit-images pt-5 py-3">
+                <div class="kenya">
+                    <!-- <img src="img/kenya.jpg"   mr-3"> -->
+                    <h3 class="summit-year">2020</h3>
+                    <p>CS African Summit in Kenya Nairobi</p>
+                </div>
+                <div class="pretoria">
+                    <!-- <img src="img/pretoria.jpg"  > -->
+                    <h3 class="summit-year">2021</h3>
+                    <p>CS African Summit in Pretoria South Africa</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+            <!-- partners section -->
+            <section class="partners mb-3 py-5 nonee">
+                <div class="partners-header">
+                    <h3 class="partner">Partner</h3>
+                    <hr class="hr-line">
+                </div>
+        
+                <div class="partner-imgs py-4">
+                    <img src="img/google.png" alt="" class="partner-img">
+                    <img src="img/mozilla.png" alt="" class="partner-img">
+                    <img src="img/ibm.png" alt="" class="partner-img">
+                    <img src="img/flutter.png" alt="" class="partner-img">
+                    <img src="img/airbnb.png" alt="" class="partner-img">
+                </div>
```

Comment:
- [ ] Kindly add `alt` so that whenever the images are not showing it will display the `alterantive` text 😄 , It will make your work more professional 👍